app/views/cases/new.html.slim

```diff
@@ -2,7 +2,7 @@
   = t('.heading')
 
 - content_for :sub_heading
-  = t4c(@case, '', 'sub_heading', case_type: @case.pretty_type)
+  = get_sar_heading(@case)
```

Comment:
You probably want to name this something more generic, as it is getting the heading for all case types.